### DISCUSSION TITLE
keep cursor position when scrolling

### DISCUSSION THREAD

**Discussion by hashibit** [helix-editor/helix#718] [Ideas]
Hi, when scrolling using `C-d` `C-u`, the cursor is now sticking with same line number, which makes the cursor jump  back and forth in window.
How about add a config option(or even make it default) to set the cursor stay in same position relative to window.
Just like vim's `C-d` `C-u`.

**Comment by zhaojiangbin** [comment]
I'd like to see the cursor to go back to exactly the same place before scrolling starts. During scrolling, the cursor should keep its position fixed to either the window or the buffer. It does neither right now.

**Comment by thomasaarholt** [comment]
I'd quite like to be able to toggle keeping the cursor in place on the window while scrolling with `j` and `k`. For me at least, its easier on the brain to read like that.
